openage/opus/setup.py

```diff
@@ -0,0 +1,14 @@
+# this file should be removed in the future.
+# asdf
+from distutils.core import setup
+from distutils.extension import Extension
+from Cython.Build import cythonize
+
+setup(
+    name = 'opusencoder',
+    ext_modules = cythonize([
+        Extension("opusenc",["opusenc.pyx"],
+                   libraries=["ogg","opus"])
+        ])
+)
+
```

Comment:
I tried, but `cimport ogg` fails. ogg.pxd is not found. even though it is in the same directory.